ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -266,20 +276,44 @@ export default class SelectHardware extends Component {
         flexDirection={FlexDirection.Column}
         alignItems={AlignItems.center}
       >
-        {this.state.selectedDevice === HardwareDeviceNames.ledger && (
-          <Box>
-            <BannerAlert
-              marginTop={6}
-              title={this.context.t(
-                'ledgerMultipleDevicesUnsupportedInfoTitle',
-              )}
-            >
-              {this.context.t(
-                'ledgerMultipleDevicesUnsupportedInfoDescription',
-              )}
-            </BannerAlert>
-          </Box>
-        )}
+        {this.state.selectedDevice === HardwareDeviceNames.ledger &&
```

Comment:
Very nit: but would have been nice to put this in a `const isLedgerDevice = ...`